index.html

```diff
@@ -878,7 +878,7 @@ <h4>createDisclosureData</h4>
                 </li>
                 <li>
 If |featureOption| equals `"anonymous_holder_binding"`,
-set `bbsProof` to the value computed by the `ProofGen` procedure from
+set `bbsProof` to the value computed by the `BlindProofGen` procedure from
```

Comment:
There's an inconsistency here between referring to the function name from the appropriate BBS spec or referring to the title of the procedure section in that spec. Here we use the function name (`BlindProofGen`) and below we use the section title `Calculate Pseudonym` and `Hidden PID Proof Generation with Pseudonym`. We should pick one of these and me consistent.

Reply:
@dlongley yes, when the draft IETF specs didn't have a function name (pseudonym signing) I used section names. Then I dropped the parenthesis after function names, i.e., `BlindProofGen()` is actually a function that is defined. Ideas/preferences for consistency?

Reply:
I think I'd prefer the function name because it's shorter and more specific for implementers.

Reply:
Bumping on this -- it can be handled in another PR if necessary, but we should get consistency at some point.

Reply:
Consistency is good. Clarity is better.

Is `BlindProofGen` a function or a procedure?
Is `Calculate Pseudonym` a procedure or a function?

I believe I've seen both procedures and functions (elsewhere) named with Pascal case. I think I've only seen multi-word names on procedures, not functions.

I think including the `()` helps clarify that the thing being discussed is a function, as I don't think I've ever seen a `()` suffix on a procedure name.

All of which to say — I would prefer to keep the `()` and use Pascal- or camel-case to name functions, and to use multi-word names for procedures, and to always be explicit about `function` calls vs `procedure` runs.

Reply:
@Wind4Greg have @TallTed's issues been addressed? I'd like to merge this PR, but not until we've addressed @TallTed's concerns.

Reply:
Hi  @msporny not  quite sure what to do  about @TallTed's concerns some of  the  inconsistent naming comes  from the IETF  drafts. I'm not sure we differentiate  between  `functions` and `procedures` as terms.  If  there is  something  concrete  that needs to be changed  happy  to  update.

Reply:
We need to be consistent with what we call these things. As someone not familiar with some of this new text, I'd question what the difference is between a "function" and a "procedure". I'd be worried that I am missing something important when seeing those words if they're used interchangeably. IOW, we should pick one and be consistent in its usage. I suggest we use the word "algorithm" throughout all of the specifications as I believe that's what we do throughout the other specifications.

I try to avoid the use of the word "function" because some developers believe it to be referring to a software library call.

I avoid the word "procedure" because readers believe it's a loosely defined set of tasks where the implementation details are unimportant.

I use the word algorithm because it doesn't have the same connotations as "function" nor "procedure" -- it is more abstract than a function but less abstract than a procedure.

Can we update ALL of the references to refer to "algorithms" and NOT use the `()` notation because that implies a concrete function (which the IETF specifications are not).